internal/app/api/v1/handlers.go

```diff
@@ -57,17 +51,22 @@ func (s *TestkubeAPI) AuthHandler() fiber.Handler {
 // InfoHandler is a handler to get info
 func (s *TestkubeAPI) InfoHandler() fiber.Handler {
 	apiContext := contextOSS
-	if os.Getenv(cloudApiKeyEnvName) != "" {
+	if s.proContext != nil && s.proContext.APIKey != "" {
```

Comment:
shouldn't be contextCloud also renamed below?

Reply:
this PR is only about the environment variables, the context is still called `cloud`. We should probably create a follow-up PR to rename the contexts as well

Reply:
ok